webdriver/tests/bidi/browsing_context/capture_screenshot/__init__.py

```diff
@@ -1,22 +1,15 @@
 from webdriver.bidi.modules.script import ContextTarget
 
+from ... import get_viewport_dimensions, get_device_pixel_ratio
```

Comment:
Please sort the imports alphabetically. We don't have linting for tests (yet) but this would fail.